ansible_collections/arista/avd/roles/eos_designs/templates/overlay/ebgp/address-family-evpn.j2

```diff
@@ -7,6 +7,7 @@
       window: {{ evpn_hostflap_detection.window | arista.avd.default(180) }}
       threshold: {{ evpn_hostflap_detection.threshold | arista.avd.default(5) }}
       enabled: {{ evpn_hostflap_detection.enabled | arista.avd.default(true) }}
+      expiry_timeout: {{ evpn_hostflap_detection.expiry_timeout | arista.avd.default }}
```

Comment:
for future Major release let's also provide best practices for what default timeout should be